oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtGrantedAuthoritiesConverter.java

```diff
@@ -50,11 +53,16 @@
 	public Collection<GrantedAuthority> convert(Jwt jwt) {
 		Collection<GrantedAuthority> grantedAuthorities = new ArrayList<>();
 		for (String authority : getAuthorities(jwt)) {
-			grantedAuthorities.add(new SimpleGrantedAuthority(DEFAULT_AUTHORITY_PREFIX + authority));
+			grantedAuthorities.add(new SimpleGrantedAuthority(authorityPrefix + authority));
 		}
 		return grantedAuthorities;
 	}
 
+	public void setAuthorityPrefix(String authorityPrefix) {
```

Comment:
Will you please add some simple JavaDoc here?

Reply:
will add the suggested javadoc